lib/stb_image.ex

```diff
@@ -103,6 +105,202 @@ defmodule StbImage do
         "unsupported tensor shape: #{inspect(shape)} (expected height-width-channel)"
       )
 
+  @compile {:no_warn_undefined, Kino}
+
+  @default_kino_render_encoding Application.compile_env(:stb_image, :kino_render_encoding, :png)
+  @doc """
+  Get preferred image encoding when rendering in Kino.
+
+  Default value is `Application.compile_env(:stb_image, :kino_render_encoding, :png)`.
+  """
+  @spec kino_render_encoding(atom()) :: term()
+  def kino_render_encoding(encoding \\ nil)
+
+  def kino_render_encoding(nil) do
+    Process.get(:stb_image_kino_render_encoding, @default_kino_render_encoding)
+  end
+
+  def kino_render_encoding(encoding) when encoding in [:png, :jpg, :jpeg] do
+    encoding =
+      if encoding == :jpeg do
+        :jpg
+      else
+        encoding
+      end
+
+    Process.put(:stb_image_kino_render_encoding, encoding)
+  end
+
+  def kino_render_encoding(unsupported_encoding) do
+    raise RuntimeError, "Encoding #{inspect(unsupported_encoding)} is not supported."
+  end
+
+  @default_kino_render_max_size Application.compile_env(
+                                  :stb_image,
+                                  :kino_render_max_size,
+                                  {8192, 8192}
+                                )
```

Comment:
I believe we can validate these at compilation time. Then we can move the attribute inside the `defimpl` module, this way we don't need to define functions and we don't need to change the process dictionary. :)